modules/prebidServerBidAdapter.js

```diff
@@ -193,6 +193,8 @@ export function PrebidServer() {
   /* Prebid executes this function when the page asks to send out bid requests */
   baseAdapter.callBids = function(s2sBidRequest, bidRequests, addBidResponse, done, ajax) {
     const isDebug = !!getConfig('debug');
+    const app = !!getConfig('app');
```

Comment:
are pubs setting `app` and `device` in the config?

Reply:
The use case here is using Postbids in an inapp environment via DFP. Nothing is explained on prebid about passing the app and device data yet so using the config seem the most relevant and logical way of doing it. 

On the server code (go) all is already done to be able to read this fields and use in requests to bidders.

Reply:
yeah that makes sense. If `app` and `device` are ORTB objects (and I think they are / will be soon) based on the new ORTB endpoint, then yes I think it makes sense to put these into the config as opposed to the S2S setting object. These can be used universally then vs just S2S.